decidim-proposals/app/controllers/decidim/proposals/admin/participatory_texts_controller.rb

```diff
@@ -47,7 +47,7 @@ def import
         def update
           enforce_permission_to :manage, :participatory_texts
 
-          form_params = params.require(:preview_participatory_text).permit!
+          form_params = params.require(:preview_participatory_text)
```

Comment:
This was no necessary as far as I could see (no broken specs after removing this)